src/components/05_pages/Content/Content.js

```diff
@@ -89,6 +89,10 @@ class Content extends Component {
     contentTypes: [],
     status: null,
     sort: { path: 'changed', direction: 'DESC' },
+    page: {
+      offset: 0,
+      limit: 5,
```

Comment:
Should we show 50 items per page which is default in the current content listing?